bin/create-webextension

```diff
@@ -9,4 +9,6 @@ if (!process.argv[2]) {
   process.exit(1);
 }
 
-require("..").main(process.argv[2]);
+require("..").main(process.argv[2])
+  .then(console.log)
+  .catch(err => console.log(err.message));
```

Comment:
@saintsebastian This is definitely better for being able to use the module as a library but we still need to make the following tweaks:

- we want to be sure that the CLI command exits with an "error exit code" (e.g. using process.exit explicitly) when there we catch an error here
- we want to be sure that we also produce a stack trace when the caught error is not expected (which currently
  means "anything but an 'EEXIST' error"), and we can do it by just re-throwing the caught error
- The expected errors (currently only the one above) should not print a stack trace and they should be possibly be
  printed using the color red

To achieve these behaviors we can use the same strategy that we are using in `web-ext`:
- define a custom UsageError class, and throw an instance of this class for the "expected" errors
- then, in the bin wrapper, check for the errors that are `instanceof` the UsageError and change the behavior
  based on it
- just re-throw the "unexpected" errors, so that they will be logged with the stack trace in the console